Collector/Formatter.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+namespace Http\HttplugBundle\Collector;
+
+use Exception;
+use Http\Client\Exception\HttpException;
+use Http\Client\Exception\TransferException;
+use Http\Message\Formatter as FormatterInterface;
+use Psr\Http\Message\RequestInterface;
+use Psr\Http\Message\ResponseInterface;
+
+/**
```

Comment:
can you please add a class phpdoc comment here that this class is a decorator for the message formatter that also formats exceptions?